graphics/netpbm/Portfile

```diff
@@ -180,8 +180,12 @@ destroot {
     # Not using fs-traverse to call file attributes because that doesn't
     # handle symlinks. Not using chown proc because that doesn't support
     # changing the group: https://trac.macports.org/ticket/55332
-    #chown ${destroot} ${install.user} ${install.group}
-    system "chown -R '${install.user}':'${install.group}' ${destroot}"
+    # Do not try to fix if not running as root; ownership is already correct.
+
+    if {$env(USER) == "root"} {
```

Comment:
String comparisons should use `eq` not `==`.

Should this use `${install.user}` instead of `$env(USER)`?

Reply:
Yes.  I made  both suggested changes.